cf/api/app_summary.go

```diff
@@ -150,3 +151,16 @@ func (repo CloudControllerAppSummaryRepository) GetSummary(appGuid string) (summ
 
 	return
 }
+
+func (repo CloudControllerAppSummaryRepository) GetSpaceSummaries(spaceGuid string) (apps []models.Application, apiErr error) {
+	resources := new(ApplicationSummaries)
```

Comment:
usually the form: `ApplicationSummaries {}` is used to create new struct. I guess no harm done but might be good to follow the current style.